packages/rocketchat-theme/client/imports/base.css

```diff
@@ -4745,13 +4745,13 @@ a + br.only-after-a {
 }
 
 .embedded-view {
-	& .flex-tab-bar {
-		display: none;
-	}
-
 	& .messages-container {
 		border-width: 0;
 
+		& .flex-tab-container {
```

Comment:
If the `flex-tab-container` is a child of `messages-container` you don't need the & there